content/mri_diffusion/bnu1.yaml

```diff
@@ -1,4 +1,5 @@
 name: BNU1
+csv: https://mrneurodata.s3.amazonaws.com/data/fmri/covariates/BNU1_phenotypic_data.csv
```

Comment:
FYI, these yaml files were generated by running `misc\m2g_script.py`  I believe at the time, this particular CSV file didn't exist which is why it was missing.

Reply:
coolio